bears/natural_language/WriteGoodLintBear.py

```diff
@@ -0,0 +1,64 @@
+from coalib.bearlib.abstractions.Linter import linter
+
+
+@linter(executable='write-good',
+        output_format='regex',
+        output_regex=r'(?P<message>.*)\s*on\s*line\s*(?P<line>\d+)\s*at\scolumn'
+                      '\s*(?P<column>\d+)'
+        )
+class WriteGoodLintBear:
+    """
+    Lints the text files using ``write-good`` for improving proses.
+    https://github.com/btford/write-good
```

Comment:
this is so plain and directly.
Try something like:
``For more information, use LINK.``